### PR TITLE
[Feature] reduce number of institutions so arrows can be seen [OSF-6478]

### PR DESCRIPTION
**Pull request by leb2dg**
## Purpose

Currently the arrows on the carousel are not highly visible if there are more than six institutions displayed.

screen shot 2016-06-09 at 11 13 56 am

## Changes

* reduce the amount of institutions displayed at a time to 5 if there are more than 6 institutions
* center institutions

More than six:
screen shot 2016-06-10 at 3 02 15 pm
screen shot 2016-06-10 at 3 05 43 pm

Six or less:
screen shot 2016-06-09 at 11 07 33 am

## Side effects

Depending on the `CAROUSEL_WIDTH`, the padding on the control arrows in `carousel.js` may have to be adjusted. The alignment of the arrows is correct for column widths of 2/carousel widths of 5 or 6.

## Ticket

https://openscience.atlassian.net/browse/OSF-6478

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Is this verified with the product team? Those arrows are very tiny; I'm not sure I would have noticed them if I weren't looking for them, but if Product is good with it, that's cool.

**Comment by brianjgeiger** [comment]
I chatted with @saradbowman and she agreed that the arrows were a bit small. Maybe something 1/4 the height of the icons or thereabouts, and possibly a bit heavier weight as well?

**Comment by brianjgeiger** [comment]
Code looks good. 🐧 Larger arrows, and I'd like to confirm that it looks good over the entire responsive range.

**Comment by leb2dg** [comment]
Thanks for checking! I'll increase the size/weight and update the screenshots.

**Comment by saradbowman** [comment]
+1! :)

**Comment by leb2dg** [comment]
The arrow sizes have been increased but the vertical placement of the arrow is not exactly center depending on the screen width. This has been okayed by @sloria for this round of improvement on the carousel.

**Comment by brianjgeiger** [comment]
Looks great. 🎉 🐧 🎉

File: website/static/js/components/carousel.js [hunk 1]
```diff
@@ -8,13 +8,19 @@ var m = require('mithril');
 // We need to use a CSS ID in order for the controls to work
 var CAROUSEL_ID = '__carousel';
 
+var controlStyle = {
```
Comment (sloria) [inline website/static/js/components/carousel.js]: 👍

**Comment by sloria** [comment]
Yep, this is ready to go. Will merge for next improvements release.

**Pull request merged**